examples/cutplate.py

```diff
@@ -84,7 +84,7 @@ def conv( degree=1, nrefine=4 ):
 
 def unittest():
 
-  retvals = main( nelems=None, degree=2, maxrefine=2, plots=False, solvetol=0 )
+  retvals = main( nelems=None, degree=1, maxrefine=2, plots=False, solvetol=0 )
```

Comment:
Yeah, it's just to suppress a warning, thought I'd sneek it in. But I'll split it out then. Purist :)